dev/docker-compose.yaml

```diff
@@ -60,8 +60,7 @@ services:
       OIDC_TOKEN_URI: "https://keycloak.${BASE_DOMAIN:-localtest.me}/auth/realms/fEMR/protocol/openid-connect/token"
       OIDC_TOKEN_INTROSPECTION_URI: "https://keycloak.${BASE_DOMAIN:-localtest.me}/auth/realms/fEMR/protocol/openid-connect/token/introspect"
       # TODO change to INTERNAL_FHIR_API
-      # TODO switch to external API (requiring auth)
-      MAP_API: "http://fhir-internal:8080/fhir/"
+      MAP_API: 'https://fhirwall.${BASE_DOMAIN:-localtest.me}/fhir/'
```

Comment:
Does cosri-patientsearch need to set the required `Authorization` request headers (to include the JWT) etc?

Reply:
I believe that is being done correctly?
https://github.com/uwcirg/cosri-patientsearch/blob/develop/patientsearch/logserverhandler.py#L24

Reply:
Sorry - I didn't get what you were asking the first time.

Fortunately, we're already doing that in cosri-patientsearch, see the `auth` parameter in https://github.com/uwcirg/cosri-patientsearch/blob/develop/patientsearch/models/sync.py#L68

Reply:
Awesome, thanks much! Couldn't remember if/when we did that work